.github/workflows/ci.yml

```
@@ -99,16 +99,67 @@
+
+      - name: Download build files
+        uses: actions/download-artifact@v5.0.0
+        with:
+          name: build-artifacts
+
+      - name: Setup local npm registry
+        run: ./scripts/setup-registry.sh
+
+      - name: Install test package dependencies
+        run: |
+          npm install 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)